moses/metrics/utils.py

```diff
@@ -138,7 +138,7 @@ def compute_scaffold(mol, min_rings=2):
 
 def average_agg_tanimoto(stock_vecs, gen_vecs,
                          batch_size=5000, agg='max',
-                         gpu=-1):
+                         gpu=-1, p=1):
```

Comment:
it can be useful to add assert on p != 0 and also documentation line to this parameter